src/sentry/api/endpoints/group_tombstone.py

```diff
@@ -11,6 +12,7 @@
 
 @region_silo_endpoint
 class GroupTombstoneEndpoint(ProjectEndpoint):
+    owner = ApiOwner.HYBRID_CLOUD
```

Comment:
```suggestion
    owner = ApiOwner.ISSUES
```

group tombstones are part of discard and delete.